examples/colors/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "gtk4-rs-examples"
+version = "0.1.0"
+authors = ["The gtk-rs Project Developers"]
+edition = "2021"
+
+[dependencies]
+chrono = "0.4.19"
+once_cell = "1.5"
+
+glium = { version = "0.30.1", optional = true, default-features = false }
+epoxy = { version = "0.1.0", optional = true }
+libloading = { version = "0.7.0", optional = true }
```

Comment:
You're not using any of these dependencies other than `once_cell`

Reply:
There should be no need for a separate cargo.toml like any of the other examples